environ/src/environ/core.cljs

```diff
@@ -0,0 +1,10 @@
+(ns environ.core
+  (:require [environ.util :as util]))
+
+(defonce ^{:doc "A map of environment variables."}
+  env
+  (->> js/process.env
+       js/Object.keys
+       js->clj
+       (map (fn [k] [(util/keywordize k) (aget js/process.env k)]))
```

Comment:
Why did you do it this way? Why not just `js->clj` the whole map and do it the same way as the Clojure code for `read-system-props` and `read-system-env`?